boards/px4/fmu-v5x/init/rc.board_sensors

```diff
@@ -55,17 +55,20 @@ else
 
 fi
 
+# External compass on GPS1/I2C1 (the 3rd external bus): standard Holybro Pixhawk 4 or CUAV V5 GPS/compass puck (with lights, safety button, and buzzer)
+ist8310 -X -b 1 -R 10 start
```

Comment:
Is I2C1 the special 10 pin connector on all v5x variants?

Reply:
> Is I2C1 the special 10 pin connector on all v5x variants?

I believe so